apps/central-scan/integration-testing/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_VX_SKIP_SCAN_ELECTION_HASH_CHECK=TRUE
+REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=TRUE
```

Comment:
We're generating the ballot package on the fly (which is better for maintainability than having a fixture we need to regenerate) so we don't have a signature.